config/spinnaker.yml

```diff
@@ -212,9 +212,13 @@ providers:
     # set enabled and provide primary credentials for deploying.
     # Enabling azure is independent of other providers.
     enabled: false
-    defaultRegion: West US
+    defaultRegion: westus
     primaryCredentials:
       name: my-azure-account
+      clientId: my-client-id
```

Comment:
I'm not familiar with Azure yet -- are these valid values or placeholders? If they are not valid, then I think you want them to be empty (even if empty is not valid either) and treat empty values as being undefined. Otherwise, if these are valid as is, then that's fine.